fe/fe-core/src/main/java/com/starrocks/qe/HDFSBackendSelector.java

```diff
@@ -82,7 +82,7 @@ public class HDFSBackendSelector implements BackendSelector {
     private final boolean forceScheduleLocal;
     private final boolean shuffleScanRange;
     private final int kCandidateNumber = 3;
-    private final int kMaxImbalanceRatio = 3;
+    private final double kMaxImbalanceRatio = 1.1;
```

Comment:
Could you elaborate a little bit the reason of choosing this value

Reply:
Done